tests/test_empirical_orthogonal_functions.py

```diff
@@ -626,19 +626,23 @@ def test_save_all_eofs_to_dir(tmp_path):
     lat = np.array([-10., 0., 10.])
     long = np.array([0., 5.])
     eofs = []
-    no_leap = False
+    no_leap_years = False
     for doy in range(1, 367):
         eof1 = np.array([1, 2, 3, 4, 5, 6]) * doy
         eof2 = np.array([10, 20, 30, 40, 50, 60]) * doy
         eofs.append(eof.EOFData(lat, long, eof1, eof2))
-    target = eof.EOFDataForAllDOYs(eofs, no_leap)
+    target = eof.EOFDataForAllDOYs(eofs, no_leap_years)
 
     print(tmp_path)
 
     errors = []
     # ToDo: (Sarah): Did you comment out the lines below and change the test? Could you give me some hints on the reason?
     # In any case, I had to indent the two lines start with if "non-existent directory" to make it work properly.
     # Maybe you can double check all the changes here?
+
+    # I don't remember making these changes, but when I try running the test cases on the master branch, this case fails in 
+    # the way you're describing. The indents you made here allow the test to pass in here, for me. - Sarah
+
```

Comment:
Thanks for your reply. So the test of the latest version inmaster also does not work and I must have made a mistake sometine before without realizing it. That is a bit worrying and I will see, if I find the respective commit...